databox/client/lib/DataboxWrapper.ts

```diff
@@ -11,6 +11,7 @@ export default class DataboxWrapper<TInput = IBasicInput, TOutput = any>
 {
   public static defaultExport: DataboxWrapper;
 
+  public readonly module = '@ulixee/databox';
```

Comment:
maybe not necessary for pr, but should we pull from package.json?

Reply:
Yes. It's now pulling from package.json.